app/server/src/main/scala/org/bitcoins/server/DLCRoutes.scala

```diff
@@ -104,5 +104,15 @@ case class DLCRoutes(dlcNode: DLCNodeApi)(implicit system: ActorSystem)
           }
         }
       }
+
+    case ServerCommand("offer-send", arr) =>
+      withValidServerCommand(OfferSend.fromJsArr(arr)) {
+        case OfferSend(offerHash, address) =>
```

Comment:
This should take offerTLV, message, and peerAddress. We do not want to send out of IncomingOffers